docs/module/stp.md

```diff
@@ -28,6 +29,7 @@ MSTP/RSTP ports fall back to regular STP upon receiving a plain STP BPDU.
 ## Global Parameters
 
 * **stp.protocol** (one of stp, mstp, rstp or pvrst) -- Global STP flavor to run on supporting nodes, default **stp**
+* **stp.stub_port_type** (one of 'normal','edge','network','auto' or 'none') -- Port type to configure on ports with only hosts connected, default **'edge'**
```

Comment:
Do you think it's a good idea to assume everyone wants "portfast" on the edge ports as the "don't care about this" value?

Reply:
Changed it to ```none```